Src/Newtonsoft.Json-for-Unity.Tests/Assets/SystemWebPolyfill/HttpUtility.cs

```diff
@@ -0,0 +1,670 @@
+        /// <param name="output">The TextWriter output stream containing the decoded string. </param>
+        public static void HtmlDecode(string s, TextWriter output) 
+        {
+            if (output == null) {
+                throw new ArgumentNullException ("output");
+            }
+                
+            if (!String.IsNullOrEmpty (s)) {
+                HttpEncoder.Current.HtmlDecode (s, output);
+            }
+        }
+
+        public static string HtmlEncode (string s)
+        {
+            if (s == null)
+                return null;
+            
+            using (var sw = new StringWriter ()) {
+                HttpEncoder.Current.HtmlEncode (s, sw);
+                return sw.ToString ();
+            }
+        }
+        
+        /// <summary>
+        /// HTML-encodes a string and sends the resulting output to a TextWriter output stream.
+        /// </summary>
+        /// <param name="s">The string to encode. </param>
+        /// <param name="output">The TextWriter output stream containing the encoded string. </param>
+        public static void HtmlEncode(string s, TextWriter output) 
+        {
+            if (output == null) {
+                throw new ArgumentNullException ("output");
+            }
+                
+            if (!String.IsNullOrEmpty (s)) {
+                HttpEncoder.Current.HtmlEncode (s, output);
+            }
+        }
+        public static string HtmlEncode (object value)
+        {
+            if (value == null)
+                return null;
+
+#if !(MOBILE || NO_SYSTEM_WEB_DEPENDENCY || true)
+            IHtmlString htmlString = value as IHtmlString;
+            if (htmlString != null)
+                return htmlString.ToHtmlString ();
+#endif
+
+            return HtmlEncode (value.ToString ());
+        }
+
+        public static string JavaScriptStringEncode (string value)
+        {
+            return JavaScriptStringEncode (value, false);
+        }
+
+        public static string JavaScriptStringEncode (string value, bool addDoubleQuotes)
+        {
+            if (String.IsNullOrEmpty (value))
+                return addDoubleQuotes ? "\"\"" : String.Empty;
+
+            int len = value.Length;
+            bool needEncode = false;
+            char c;
+            for (int i = 0; i < len; i++) {
+                c = value [i];
+
+                if (c >= 0 && c <= 31 || c == 34 || c == 39 || c == 60 || c == 62 || c == 92) {
+                    needEncode = true;
+                    break;
+                }
+            }
+
+            if (!needEncode)
+                return addDoubleQuotes ? "\"" + value + "\"" : value;
+
+            var sb = new StringBuilder ();
+            if (addDoubleQuotes)
+                sb.Append ('"');
+
+            for (int i = 0; i < len; i++) {
+                c = value [i];
+                if (c >= 0 && c <= 7 || c == 11 || c >= 14 && c <= 31 || c == 39 || c == 60 || c == 62)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4755183)